{"name": "页数为\"1\"，记录数为\"0\"", "status": "broken", "statusDetails": {"message": "NameError: name 'os' is not defined", "trace": "@pytest.fixture(scope= 'session')\n    def login_init():\n        print('---用户登录操作---')\n        # 登录操作\n>       token = Login().login(NAME_PWD, get_token=True)\n\nconftest.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <libs.login.Login object at 0x000001A355885270>, user_token = None\nkwargs = {}\n\n    def __init__(self, user_token=None, **kwargs):\n>       self.data = get_yml_data(os.path.join(config_path,'apiPathConfig.yml'))[self.__class__.__name__]  # 怎么取？\nE       NameError: name 'os' is not defined\n\n..\\common\\baseApi.py:24: NameError"}, "parameters": [{"name": "title", "value": "'页数为\"1\"，记录数为\"0\"'"}, {"name": "body", "value": "{'page': 1, 'limit': 0}"}, {"name": "exp_data", "value": "{'error': 'Internal Server Error'}"}], "start": 1669971901949, "stop": 1669971901949, "uuid": "dfe9ba78-c929-4ccf-88c4-1b0ec5b777a1", "historyId": "3b90de16cc71334d07fd697d58a9db1e", "testCaseId": "8115a05f0983cc4141f41c92bacc50d1", "fullName": "testCase.test_shop.TestShop#test_shop_query", "labels": [{"name": "story", "value": "店铺的查询"}, {"name": "feature", "value": "店铺模块"}, {"name": "epic", "value": "订餐系统"}, {"name": "tag", "value": "shop_query"}, {"name": "tag", "value": "shop"}, {"name": "parentSuite", "value": "testCase"}, {"name": "suite", "value": "test_shop"}, {"name": "subSuite", "value": "TestShop"}, {"name": "host", "value": "DESKTOP-BPP8M8A"}, {"name": "thread", "value": "15352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCase.test_shop"}]}